dt-assets/js/details.js

```diff
@@ -94,6 +95,19 @@ jQuery(document).ready(function($) {
     rest_api.update_post(post_type, post_id, { [fieldKey]: fieldValues }).then((newPost)=>{
       $(`#${fieldKey}-spinner`).removeClass('active')
       post = newPost
+
+      // Safeguard against immediate update failures.
```

Comment:
@kodinkat can we clarify these comments
what is an update failure?
Is this just `//make sure a key exists for the new link field?`